### PR TITLE
Set disbaleSsh default value to true for workstation create

### PR DESCRIPTION
**Pull request by tejal29**
fixes https://github.com/hashicorp/terraform-provider-google/issues/19094

**Release Note Template for Downstream PRs (will be copied)**

```release-note:breaking-change
workstation: `host.gce_instance.disable_ssh` now defaults to true for `google_workstations_workstation_config`
```

### CODE REVIEW

**Review by c2thorn — CHANGES_REQUESTED** [review]
The errors look intermittent, but adding a default value to an existing field is a breaking change. This can happen in the 6.0.0 branch that we have incoming for the next major release, would you like to move the change there?

Also, is there an associated Github issue and/or buganizer ticket for this change?

**Comment by tejal29** [comment]
> The errors look intermittent, but adding a default value to an existing field is a breaking change. This can happen in the 6.0.0 branch that we have incoming for the next major release, would you like to move the change there?
> 
> Also, is there an associated Github issue and/or buganizer ticket for this change?

sounds good. I can move this PR there. 
There is a buganizer issue. Is it safe to link it here?

**Comment by c2thorn** [comment]
> sounds good. I can move this PR there. There is a buganizer issue. Is it safe to link it here?

yes, shortened b/<number> links are posted in our Github issues all the time

> Is this the 6.0.0 branch? [`FEATURE-BRANCH-major-release-6.0.0`](https://github.com/GoogleCloudPlatform/magic-modules/tree/FEATURE-BRANCH-major-release-6.0.0?rgh-link-date=2024-07-23T20%3A42%3A38Z)

That is correct, I can change the base branch in this PR myself. Let me do so

**Review by c2thorn — COMMENTED** [review]
This still needs an associated Github issue in 6.0.0 and an upgrade guide entry following https://googlecloudplatform.github.io/magic-modules/develop/breaking-changes/make-a-breaking-change/#make-the-change-on-feature-branch-major-release-600

File: mmv1/third_party/terraform/website/docs/guides/version_6_upgrade.html.markdown [hunk 1]
```diff
@@ -318,6 +318,12 @@ on the resource through Terraform and the default labels configured on the provi
 * The new output-only `effective_labels` field lists all of labels present on the resource
 in GCP, including the labels configured through Terraform, the system, and other clients.
 
+## Resource: `google_workstations_workstation_config`
+
+### Disable SSH to root VM is set to true by default for workstation config.
```
Comment (c2thorn) [inline mmv1/third_party/terraform/website/docs/guides/version_6_upgrade.html.markdown:323]:
```suggestion
### `host.gce_instance.disable_ssh` now defaults to true
```

File: mmv1/third_party/terraform/website/docs/guides/version_6_upgrade.html.markdown [hunk 2]
```diff
@@ -318,6 +318,12 @@ on the resource through Terraform and the default labels configured on the provi
...
+
+* `disableSSH` field is set to true when creating configs. To enable SSH, please set `disableSSH` to false.
```
Comment (c2thorn) [inline mmv1/third_party/terraform/website/docs/guides/version_6_upgrade.html.markdown:325]:
```suggestion
* `disable_ssh` field now defaults to true. To enable SSH, please set `disable_ssh` to false.
```

**Review by c2thorn — APPROVED** [review]

**Pull request merged**